### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Jamulus app](https://hosted.weblate.org/projects/jamulus/jamulus-app/).

It also includes following components:

* [Jamulus/Windows Installer](https://hosted.weblate.org/projects/jamulus/windows-installer/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by ann0see — COMMENTED** [review]
@comradekingu please approve your translations such that they don’t show up as unfinished.

**Review by jujudusud — APPROVED** [review]

**Comment by comradekingu** [comment]
@ann0see I do that so I know what I am unsure about. Mostly I do it to keep track of what source strings need fixing.

**Comment by ann0see** [comment]
@jujudusud great. It's just that it would be good to have one approver per language before it gets merged.

**Comment by comradekingu** [comment]
@ann0see 
If you want to review everything I have done, run it through auto-translation.
The quality of the source strings isn't good enough to warrant complaining about anyone else.
Removing the "needs editing" flags makes it much harder for me to change that.

**Comment by ignotus666** [comment]
@comradekingu 
What's so terribly wrong with the source strings? Could you give some examples? Sure some strings could possibly be made clearer, but you do seem to sound quite dismissive of them as a whole, as if they're so bad a proper translation isn't possible. You're the first translator to raise this as an obstacle for translating.

**Comment by ann0see** [comment]
> If you want to review everything I have done, run it through auto-translation.

No need for that. Usually we let the translators decide and ask for other potential reviewers if they exist to spot spelling mistakes etc. If you're not yet ok with some translations that's totally fine (the translator is mostly in charge of his/her own translation anyway). As soon as we reach release candidate stage, the existing strings should be "releasable".

**Comment by ann0see** [comment]
> What's so terribly wrong with the source strings?

I think there are some unclear strings - at least I find some once in a while. 
Best would be to raise an issue and list them with a suggestion + explanation what needs fixing

**Review by trebmuh — APPROVED** [review]
:+1:

**Comment by ann0see** [comment]
Review checklist:

- [ ] Translator listed in the `src/util.cpp` *optionally add link to PR or code* 
- [ ] Punctuation and spacing consistent
- [ ] Signal words consistent ("ASIO", "Buffer")
- [ ] App translations: No untranslated strings (`grep unfinished -5 src/translation/translation_$TRANSLATION*.ts`)
- [ ] App translations: Only a single `.ts` file checked in (`.qm` in addition is also OK)
- [ ] Installer translations: Passes `tools/check-wininstaller-translations.sh`

File: src/translation/translation_nb_NO.ts
Comment (ann0see) [inline src/translation/translation_nb_NO.ts:213]: I'd check how this looks in the app @comradekingu since Panorering might be too big.

File: src/translation/translation_nb_NO.ts
Comment (ann0see) [inline src/translation/translation_nb_NO.ts:735]: Good point about the typography here. Should the source string should also use `…` instead of three dots?

File: src/translation/translation_nb_NO.ts
Comment (ann0see) [inline src/translation/translation_nb_NO.ts:972]: I think Forstummet should also be all caps to reassemble the English version

File: src/translation/translation_nb_NO.ts
Comment (ann0see) [inline src/translation/translation_nb_NO.ts:2733]: That's strange: Why is it an empty string?

@hoffie probably some future translation CI script should validate that too.

File: src/translation/translation_nb_NO.ts
Comment (ann0see) [inline src/translation/translation_nb_NO.ts:2774]: As above: probably this should be all caps.

File: src/translation/translation_pt_PT.ts
Comment (ann0see) [inline src/translation/translation_pt_PT.ts:196]: Just as small comment, this doesn't need to be translated

File: src/translation/translation_es_ES.ts
Comment (ann0see) [inline src/translation/translation_es_ES.ts:139]: Did we discuss anywhere that we really translate all languages?

**Review by ann0see — COMMENTED** [review]
Thanks all! Just a few comments

File: src/translation/translation_nb_NO.ts
Comment (pljones) [reply src/translation/translation_nb_NO.ts:213]: Yes, the checkbox and label want to be as small as possible.

File: src/translation/translation_es_ES.ts
Comment (ignotus666) [reply src/translation/translation_es_ES.ts:139]: This is in the 'About' window listing the languages and their translators. I think it makes sense to translate it.

Comment (ann0see) [reply src/translation/translation_es_ES.ts:139]: Ok.

File: src/translation/translation_nb_NO.ts
Comment (pljones) [reply src/translation/translation_nb_NO.ts:735]: Using special typography should, in my view, be avoided.  I don't know how a screen reader would render the special character, for example.  Using `...` to indicate a menu item opens a new window is pretty standard, too - not following that might also cause accessibility issues.  I think it doesn't look good, either.

Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:735]: @pljones
The error here is the missing hyphen(s), and/or prepositions.
The translation is ~"Load the settings for the channel mixer", which I got from looking at the Swedish string.

The use of an ellipsis is correct, insofar as both typography and current paradigms in software go.
A screen reader will not output "dot" "dot" "dot", which is annoying and wastes time.
The ellipsis is the correct character to use, and Weblate triggers and error when this isn't the case.
If it doesn't look good for you, it is possible to change the font used.

https://develop.kde.org/hig/style/writing/placeholder/
https://developer.gnome.org/hig/guidelines/typography.html
https://wiki.amigaos.net/wiki/UI_Style_Guide_Menus#Ellipsis
https://www.stylemanual.gov.au/grammar-punctuation-and-conventions/punctuation-and-capitalisation/ellipses#do_not_use_a_string_of_full_stops
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/ellipsis-button

The ellipsis does not create issues unless UTF-8 support is missing.

File: src/translation/translation_nb_NO.ts
Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:2733]: It could be flagged "read only" in Weblate to avoid it for now.

File: src/translation/translation_nb_NO.ts
Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:2774]: I am of the opinion that nothing should ever be all caps.

https://developer.gnome.org/hig/guidelines/typography.html "Do not capitalize every letter (all caps)."
https://developers.google.com/style/capitalization (says sentence case).
https://develop.kde.org/hig/style/writing/capitalization/ doesn't mention even doing all caps.

File: src/translation/translation_nb_NO.ts
Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:213]: There is a https://docs.weblate.org/en/latest/user/checks.html?highlight=max#maximum-length-of-translation
available.
I think it is also possible to take into account how that factors into the given font used, but a sensible limit helps sort this for now.

File: src/translation/translation_nb_NO.ts
Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:972]: I don't really want to resemble the English version.
The nb_NO one is consistent, and the user needs never know the English one does anything differently.

https://www.compart.com/en/unicode/search?q=speaker#characters or something like
a crossed out person talking, a crossed out speech bubble, etc. would work. I think that is what others do.
bilde

**Comment by comradekingu** [comment]
@ignotus666 The source strings aren't too far off, but I'd rather do something about it.
This way it becomes far easier to translate Jamulus for yet others.
I don't have a lot of time, but I'll have a look and see what files need changing to update the source strings correctly.

File: src/translation/translation_pt_PT.ts
Comment (ann0see) [reply src/translation/translation_pt_PT.ts:196]: About the Unicode copyright © symbol, I agree. Probably it’s worth changing after the upcoming release.

File: src/translation/translation_nb_NO.ts
Comment (trebmuh) [reply src/translation/translation_nb_NO.ts:2733]: @comradekingu: you just seem to have forgotten to translate this string, don't you?

Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:2733]: @trebmuh 
https://hosted.weblate.org/translate/jamulus/jamulus-app/nb_NO/?checksum=bd82c5a091ec5c9c
From memory I was unsure if "%1" is a numeral or the name of a server.
Hence the question https://hosted.weblate.org/translate/jamulus/jamulus-app/nb_NO/?checksum=bd82c5a091ec5c9c
"Is this a known server (name), or is it an amount of servers?"

Comment (trebmuh) [reply src/translation/translation_nb_NO.ts:2733]: Then you decided to leave it in English?
If so, that's a better idea IMHO to leave the translation "empty" then it's still marked "unfinished" and fall back to display it in English.

**Review by ann0see — APPROVED** [review]
Approving as this must go in ASAP, before the translators not knowing or using weblate yet might get conflicts.

**Review by ignotus666 — APPROVED** [review]

**Comment by jujudusud** [comment]
We also need to add the information for the translators in the translation.md file and in each issue created.

**Comment by pljones** [comment]
Plus what ever process documents adding a new translator needs to be reviewed -- ideally so we've got an issue template we can use.

**Comment by jujudusud** [comment]
Fixes #2867

**Comment by henkdegroot** [comment]
Fixes #2870

File: src/translation/wininstaller/pt_PT.nsi
Comment (ann0see) [inline src/translation/wininstaller/pt_PT.nsi:34]: Please do not remove #asio here. It should still match the anchor on the pt site.

File: src/translation/translation_fr_FR.ts
Comment (ann0see) [inline src/translation/translation_fr_FR.ts:600]: I assume the removal of (Windows) has a reason?

File: src/translation/translation_fr_FR.ts
Comment (ann0see) [inline src/translation/translation_fr_FR.ts:1625]: PipeWire should then also be corrected in the source string.

File: src/translation/translation_fr_FR.ts
Comment (jujudusud) [reply src/translation/translation_fr_FR.ts:600]: Is it "Windows OS" only ?
If so, I will add "(Windows)".

Comment (ann0see) [reply src/translation/translation_fr_FR.ts:600]: I don’t think it‘s technically only Windows, but is most visible on Windows. If you drop windows, the English version should be updated too.

**Comment by pljones** [comment]
https://hosted.weblate.org/translate/jamulus/jamulus-app/en/?q=has:comment#comments
Can that lot get closed?

**Comment by ignotus666** [comment]
@pljoness those messages on weblate are gone now.

File: src/translation/translation_de_DE.ts
Comment (pljones) [inline src/translation/translation_de_DE.ts:3549]: `type="unfinished"` should be removed?

File: src/translation/translation_de_DE.ts
Comment (pljones) [inline src/translation/translation_de_DE.ts:3548]: Note: English should also use "&quot;" (single quoted speech mark) rather than "&apos;" (apostrophe of omission).

File: src/translation/translation_de_DE.ts
Comment (pljones) [inline src/translation/translation_de_DE.ts:3554]: Consistency: `&quot;` above, `&apos;` here.

File: src/translation/translation_de_DE.ts
Comment (pljones) [inline src/translation/translation_de_DE.ts:3553]: As above, should be "&quot;".

File: src/translation/translation_de_DE.ts
Comment (ann0see) [reply src/translation/translation_de_DE.ts:3549]: This still needs review by another translator (I'm not happy with both the translation and the source as it starts with "Or ...". It should give more context)

Comment (pljones) [reply src/translation/translation_de_DE.ts:3549]: Agreed - hm, probably my fault there:
```java
     QString strDirectoryTypeWT = "<b>" + tr ( "Directory" ) + ":</b> " +
                                  tr ( "Select '%1' not to register your server with a directory." )
                                  .arg ( DirectoryTypeToString     ( AT_NONE ) ) + "<br>" + 
                                  tr ( "Select one of the genres to register with that directory." ) + "<br>" +
                                  tr ( "Or select '%1' and specify a Custom Directory address on the "
                                       "Options tab to register with a custom directory." )
                                      .arg ( DirectoryTypeToString ( AT_CUSTOM ) ) +
```
Should we include the HTML in the English?

Comment (ann0see) [reply src/translation/translation_de_DE.ts:3549]: I think so. Yes.

Comment (pljones) [reply src/translation/translation_de_DE.ts:3549]: OK, raised #2894 just for this one.

Comment (ann0see) [reply src/translation/translation_de_DE.ts:3549]: Thanks.

**Review by jerogee — APPROVED** [review]

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
@hoffie yes, @comradekingu is missing in the utils.h file, as far as I know.

**Pull request merged**
